app/components/show-header/template.hbs

```diff
@@ -3,7 +3,8 @@
     {{nypr-a-svg className="site-header-title-logo" icon="wqxr-log-frq"}}
   </div>
   <div class="col" id="live-nyc-txt-box">
-    <p class="live-nyc-txt" data-test-element="header-tagline">LIVE FROM NYC</p>
+    <a href="https://beta-wqxr.demo.nypr.digital/listen" class="live-nyc-txt" data-test-element="header-tagline">LIVE FROM NYC
+    </a>
   </div>
```

Comment:
make this one line.

also, you might be doing this already, but the favored patters for uppercasing is to use the `text-transform: uppercase` rule instead of capitalizing the letters in the template.